src/components/LogRow/BaseRow/index.tsx

```diff
@@ -99,14 +103,16 @@ const BaseRow = forwardRef<any, BaseRowProps>((props, ref) => {
         size="small"
       />
       <Index>{lineNumber}</Index>
-      <ProcessedBaseRow
-        color={resmokeRowColor}
-        data-cy={dataCyText}
-        searchTerm={searchTerm}
-      >
-        {children}
-      </ProcessedBaseRow>
-    </StyledPre>
+      <StyledPre shouldWrap={wrap}>
+        <ProcessedBaseRow
+          color={resmokeRowColor}
+          data-cy={dataCyText}
+          searchTerm={searchTerm}
+        >
+          {bookmarked && prettyPrint ? formatPrettyPrint(children) : children}
```

Comment:
This feature only really makes sense for resmoke rows. I wonder if we should gate it and disable the option for non resmoke rows?

Reply:
If you agree then i think we can move this up a level to ResmokeRow and remove the need for BaseRow to have to know about prettyPrint

Reply:
So technically I did prevent it from working on `AnsiiRow`s, but I didn't pull it up to `ResmokeRow`... it's because I would also need to pull up the `bookmarks` and `resetRowHeightAtIndex`, which might be a little awkward.

Reply:
Ok cool can we cut a ticket to investigate refactoring this and other resmoke functionality and put them in PM-3000?

Reply:
added [EVG-18167](https://jira.mongodb.org/browse/EVG-18167) 💃